src/utilities/TypeInfo.ts

```diff
@@ -129,6 +140,10 @@ export class TypeInfo {
     return this._argument;
   }
 
+  getSignature(): Maybe<GraphQLSignature> {
```

Comment:
Alternatively we could expose a `getFragmentVariableSignature()` helper that only returns a value if we are within an argument for a fragment variable. Or we could have both the more specific and the more generic helpers.